Fix apportion abundance doc page (#6)
brandynlucca/echopop: docs/theory/apportion_abundance.md

$$ 
B_{\textrm{M}}^{k} =
\sum_{\ell} B_{\textrm{M}, \ell}^{k, \textrm{unaged}} +
\sum_{\ell, \alpha} B_{\textrm{M}, \ell, \alpha}^{k, \textrm{aged}}
\label{eq:biomass_M} \tag{1}
$$

$$
B_{\textrm{F}}^{k} =
\sum_{\ell} B_{\textrm{F}, \ell}^{k, \textrm{unaged}} +
\sum_{\ell, \alpha} B_{\textrm{F}, \ell, \alpha}^{k, \textrm{aged}}
\label{eq:biomass_F} \tag{2}
$$

The biomass estimates for all fish including both sexed and unsexed fish in the transect interval $k$ is then:

$$
B^k = B_\textrm{M}^k + B_\textrm{F}^k.
$$

The estimated abundance $\hat{N}^k$ can be back-calculated from the kriged biomass estimates using an averaged length-weight relationship $\overline{W}(\ell)$ via:

$$
\hat{N}^{k} = \frac{B^{k}}{\overline{W}(\ell)},
\label{eq:abundance} \tag{3}
$$

where $\overline{W}(\ell)$ is the length-weight regression relationship derived from the catch data. 

Similarly, $\hat{\textit{NASC}^{k}}$ can be back-calculated from the estimated abundance using the averaged differential backscattering cross-section of the $i^{\text{th}}$ stratum, $\bar{\sigma}_{bs}^i$, via:

$$
\hat{\textit{NASC}^k} = \hat{N}^k \times \bar{\sigma}_\textrm{bs}^i,
$$

> when the transect interval $k$ falls in stratum $i$. See [](stratification) for more information.

Reviewer: I see that the attention admonition concerning the strata indexing was removed. Instead of removing it entirely, would it make sense to include it here since the subsequent calculations are implicitly indexed by $i$ (or in some cases explicitly such as the proportions) sans $\overline{W}(\ell)$?

Author: I was thinking this would be covered by the 4-section refactoring suggested in #279, and there we can clearly list out in for what variables indexing using $k$ actually implies a specific stratum $i$ due to the geographical location of the $k$ interval?

```{note} 

> In Chu's Echopro implementation, both $\hat{N}_{s}^{k}$ and $\hat{N}^{k}$ are calculated using a single $\overline{W}(\ell)$ fit from **all** (male, female, and unsexed) fish samples, instead of sex-specific fits.

Reviewer: ```suggestion
Both $\hat{N}_{s}^{k}$ and $\hat{N}^{k}$ are calculated using a single $\overline{W}(\ell)$ fit from **all** (male, female, and unsexed) fish samples, instead of sex-specific fits in the original `EchoPro` MATLAB implementation.
```

Reviewer: I think that including this as a separate admonition is fine; however, the wording seems to imply a distinction with Echpop. If this were ever implemented in Echopop, we'd likely replicate the MATLAB code for consistency, no? I have no strong opinion on this one way or another, but just want to make sure we avoid implying there is a difference between `Echopop` and `EchoPro` here when they may not be one in the future.

Author: Yeah I agree with not implying either way, but am confused here -- wouldn't the changes you added make it actually different from EchoPro? since it says "instead of sex-specific fits in the original `EchoPro` MATLAB implementation"?

Reviewer: >Yeah I agree with not implying either way, but am confused here -- wouldn't the changes you added make it actually different from EchoPro? since it says "instead of sex-specific fits in the original EchoPro MATLAB implementation"?

I didn't want to change the meaning of your suggested change too much without discussing that first.

```suggestion
Both $\hat{N}_{s}^{k}$ and $\hat{N}^{k}$ are calculated using a single $\overline{W}(\ell)$ fit from **all** (male, female, and unsexed) fish samples in the original `EchoPro` MATLAB implementation.
```

\end{equation}
$$

Therefore, the total numbers of male and female aged fish of length $\ell$ and age $\alpha$ are:

$$
\begin{equation}
\begin{aligned}
    n_{\textrm{M}}^{\textrm{aged}} &= \sum_{\ell,\alpha}n_{\textrm{M},\ell,\alpha}^{\textrm{aged}} \nonumber \\
    n_{\textrm{F}}^{\textrm{aged}} &= \sum_{\ell,\alpha}n_{\textrm{F},\ell,\alpha}^{\textrm{aged}} \nonumber \\    
\end{aligned}
\label{eq:total_aged_sex} \tag{7}
\end{equation}
$$

### Number proportions

> The sex-specific numbers for unaged $\eqref{eq:total_unaged_sex}$ and aged $\eqref{eq:total_aged_sex}$ fish are then summed to calculate the total number of unaged fish($n^{\textrm{unaged}}$), aged ($n^{\textrm{aged}}$), and all ($n$) fish:

Reviewer: Perhaps 'numbers' should be replaced with 'counts'? I suppose it is not a huge deal so long as we are consistent in usage.

Author: Sounds good!

Author: Right now everywhere else we use "numbers," including in the glossary. I have no problem with changing it to "counts" if it is more clearer.

Author: oh wait, wouldn't "number proportion" then need to be "count proportion"? it sounds a little weird.

Reviewer: Aye, that's true. I think it's fine to use 'number' so long as we are consistent.

> The sex-specific numbers for unaged $\eqref{eq:total_unaged_sex}$ and aged $\eqref{eq:total_aged_sex}$ fish are then summed to calculate the total number of unaged fish($n^{\textrm{unaged}}$), aged ($n^{\textrm{aged}}$), and all ($n$) fish:

Reviewer: ```suggestion
The sex-specific numbers for unaged $\eqref{eq:total_unaged_sex}$ and aged $\eqref{eq:total_aged_sex}$ fish are then summed to calculate the total number of unaged fish ($n^{\textrm{unaged}}$), aged ($n^{\textrm{aged}}$), and all ($n$) fish:
```

#### Unaged fish

The number proportions of male and female unaged fish of length $\ell$ $\eqref{eq:total_unaged_sex_length}$ relative to the sex-specific totals of unaged fish $\eqref{eq:total_unaged_sex}$ are:

$$
\begin{equation}
\begin{aligned}

>     {r_N}_{\textrm{M},\ell}^{\textrm{unaged/unaged}} &= \frac{n_{\textrm{M},\ell}^{\textrm{unaged}}}{n_{\textrm{M}}^{\textrm{unaged}}} \nonumber \\

Reviewer: Here I think these should be $r_n$. To me this implies that these are relative proportions calculated from abundances when they are actually computed using the fish numbers/counts. Perhaps that is not a significant distinction given that these are used to apportion the derived abundances; however, I think if we use $r_N$ here then we would want to use $r_B$ (we currently do not have a subscript denoting "weight" specifically for this since we just use $r$ currently) for the weight proportions since they are used to apportion the derived biomasses.

Author: $r_n$ is fine - this is something I had not changed after changing all $N$ to $n$ in [#280](https://github.com/OSOceanAcoustics/echopop/pull/280). In the apportioning biomass page I wrote this ratio as a probability, but using a ratio is fine too.

For weight proportion, I prefer that we stay with $r_w$ instead of $r_B$ .
